### PR TITLE
GirderItemIdToVolume transform

### PR DESCRIPTION
**Pull request by zachmullen**

### CODE REVIEW

File: girder_worker/docker/transforms/girder.py [hunk 1]
```diff
@@ -168,6 +168,57 @@ def _repr_model_(self):
...
+            os.makedirs(path)
+        except OSError as e:
+            if e.errno != errno.EEXIST:
+                raise
+
+        return self._item_id, path
+
+    def transform(self, **kwargs):
+        self._volume.transform(**kwargs)
+        dir = self._volume.host_path
```
Comment (kotfic) [inline girder_worker/docker/transforms/girder.py]: can we name this something other than built-in?

**Comment by kotfic** [comment]
Other than the `dir` thing this all looks good to me,  I'd like to see an integration test with an item and two files that makes sure both files are downloaded. You might want to make a `test_item_with_files` fixture

**Comment by zachmullen** [comment]
@kotfic my integration test is now succeeding, but a different one is failing, and I have no idea why, any thoughts?

**Comment by kotfic** [comment]
First guess is it has something to do with the  `test_multi_file_item` fixture being set to function scope,  but not cleaning up the item after the `yield`

**Comment by zachmullen** [comment]
All set. The extraneous changes on here are due to chance breakages in two upstream libraries where we always install the latest version: click and flake8.

**Pull request merged**